lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -150,6 +150,8 @@ def purchase(amount, payment_method, options = {})
         add_submerchant_id(post, options)
         add_stored_credential(post, options)
         add_processor(post, options)
+        add_crypto_currency_type(post, options)
+        add_transaction_type(post, options)
```

Comment:
This addition of `add_transaction_type` is redundant since it's called in `add_stored_credential`